float32 meas_voltage
float32 prog_voltage
float32 meas_current
float32 prog_current
bool sr_constant_volt
bool sr_constant_curr
bool sr_no_fault
bool sr_fault
bool sr_auto_start
bool sr_fold_enabled
bool sr_local_mode
bool fr_ac_fail
bool fr_ovet_temp
bool fr_foldback
bool fr_over_volt_prot
bool fr_shutt_off
bool fr_output_off
